map_view/display-layer-view-state/src/main/resources/display_layer_view_state/style.css

```
@@ -1,27 +1,18 @@
-.panel-region .label {
-  -fx-text-fill: white;
-}
-
-.label {
-  -fx-text-fill: black;
-}
-
-.slider .axis {
-    -fx-tick-label-fill: white;
+.panel-region {
+  -fx-padding: 10;
+  -fx-background-color: rgba(0, 0, 0, 0.5);
+  -fx-max-width: 400;
+  -fx-max-height: 120;
+  -fx-spacing: 6;
+  -fx-alignment: center;
 }
 
-.range-slider .axis {
-    -fx-tick-label-fill: white;
-}
-
-.panel-region .check-box {
-   -fx-text-fill: white;
-}
-
-.panel-region .radio-button {
-   -fx-text-fill: white;
+.panel-label {
+  -fx-text-fill: white;
+  -fx-font-size: 16px;
+  -fx-text-alignment: center;
```

It looks like this might be older code, but since we've got this file open there's a bit of stuff we might be able to improve here. There's no need to specify a unique CSS rule for a label that sits within the `panel-region` which you've defined above.
The old rule in this file will apply to any `Label` sitting within the node that get's the style class `panel-region`.
You can then get rid of `layerViewStatusLabel.getStyleClass().add("panel-label");` in the main code, since there's no need to define a specific class for labels inside the panel anymore.

So you could do:
```suggestion
.panel-region .label {
  -fx-text-fill: white;
  -fx-font-size: 16px;
  -fx-text-alignment: center;
```